mix.exs

```diff
@@ -90,6 +90,9 @@ defmodule ExDoc.Mixfile do
           ExDoc.ModuleNode,
           ExDoc.TypeNode
         ]
+      ],
+      group_modules_by_nesting: [
+        ExDoc.Formatter
```

Comment:
I don't think we need this for ExDoc itself, as our module names are not that long. :)

Reply:
Oops, definitely not: it was a quick way to test during development and snuck through when committing :(